packages/react-server/src/ReactFlightServer.js

```diff
@@ -441,15 +441,11 @@ function RequestInstance(
   onAllReady: void | (() => void),
   onFatalError: void | ((error: mixed) => void),
 ) {
-  if (
-    ReactSharedInternals.A !== null &&
-    ReactSharedInternals.A !== DefaultAsyncDispatcher
-  ) {
-    throw new Error(
-      'Currently React only supports one RSC renderer at a time.',
-    );
+  const previousAsyncDispatcher = ReactSharedInternals.A;
+  if (previousAsyncDispatcher === null) {
+    ReactSharedInternals.A = DefaultAsyncDispatcher;
```

Comment:
This isn’t really good enough even as an intermediate step to land since it’ll silently cover up that caches etc don’t work.